tests/lib/screenshot-testing/support/chai-extras.js

```diff
@@ -42,6 +42,13 @@ function getPageLogsString(pageLogs, indent) {
     return result;
 }
 
+function logErrorMessage(message, indent) {
+    var indent = "     ";
+    console.log("\n\n" + indent + 'This message usually belongs to the next test');
+    message = message.replace(/\n/g, "\n" + indent);
+    console.log(indent + message);
+}
```

Comment:
I'll update. Have to do it in `app.js`, `app` seems to be not yet created when page-renderer is loaded etc but that shouldn't be a problem. Hope it will still log for all cases